packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -280,9 +281,30 @@ class EditableText extends StatefulWidget {
   /// The type of keyboard to use for editing the text.
   final TextInputType keyboardType;
 
+  /// The type of action button to use with the soft keyboard.
+  final TextInputAction textInputAction;
+
   /// Called when the text being edited changes.
   final ValueChanged<String> onChanged;
 
+  /// Called when the user submits editable content (ex: user presses the "done"
```

Comment:
s/ex/e.g./

Reply:
can you elaborate on what you're requesting here?

Reply:
the syntax `s/pattern/replacement/` is used in various unix programs (sed, vim, perl, etc) to mean "replace the 'pattern' part with 'replacement'". So in this case, I meant "replace the "ex" (in the parenthetical) with "e.g."".